### PR TITLE
French translation of Getting Started

### PR DESCRIPTION
**Pull request by ghost**
Checkout the [issue](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/7244) for this pull request to track progress.

### CODE REVIEW

**Comment by thejonathangraham** [comment]
Please refer to this issue for the french translation of "getting started" - https://github.com/FreeCodeCamp/FreeCodeCamp/issues/7244

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -160,6 +159,81 @@
           "Si completas todas las 2,080 horas de desafíos y proyectos, obtendrás un Certificado de Desarrollo de Pila Completa.  Te ofreceremos gratuitamente una práctica en entrevistas para desarrolladores. Incluso ofrecemos un directorio de trabajos donde los empleadores contratan específicamente campistas que han recibido certificaciones de Free Code Camp.",
           ""
         ]
+      ],
+      "nameFr": "Apprenez comment gratuites Code Camp de Travaux",
```
Comment (fnonga) [inline seed/challenges/00-getting-started/getting-started.json]: "nameFr": "Apprenez comment Free Code Code fonctionne",

**Comment by BerkeleyFalse** [comment]
@thejonathangraham updated the pull request.

**Comment by bugron** [comment]
@thejonathangraham please, read our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) very carefully and then open a new PR with all changes in it.

**Pull request closed without merging**